### ISSUE TITLE
Contributing guidelines?

### ISSUE THREAD

**Issue by DynamicSquid** [JeffreyLin21/CCC-Solutions#1]
Hello, this seems interesting!

What are the contributing guidelines?

**Comment by JeffreyLin21** [comment]
Just added some new solutions, feel free to take or add whatever you want.

**Comment by DynamicSquid** [comment]
Gotcha, also is there a preferred code style when adding C++ solutions? Or anything works as long as it's readable?

**Comment by JeffreyLin21** [comment]
Nope, not really, you can always fix it later if you want, but as long as it works it'll be just fine.
